hw9/exceptions/ArraySizeException.java

```diff
@@ -0,0 +1,7 @@
+package org.hillel.hw9.exceptions;
+
+public class ArraySizeException extends IndexOutOfBoundsException {

```

Comment:
`IndexOutOfBoundsException` is not really a good candidate, because the size-related error is not really about an index, it's about a size of a matrix and its rows. So would better to extends from `RuntimeException`